templates/share/site_header.html

```diff
@@ -82,7 +82,7 @@
                         <strong>Recent</strong>
                         <ul class="item-list">
                             {% for wlv in user|get_workbooks_this_user|slice:':5' %}
-                            <li class="menu-item" title="{{ wlv.name|wrap_text }}"><a href="{% url 'workbook_detail' wlv.id %}">{{ wlv.name }}</a></li>
+                            <li class="menu-item" title="{{ wlv.name|wrap_text }}"><a style="display: inline-block;" href="{% url 'workbook_detail' wlv.id %}">{{ wlv.name }}</a></li>
```

Comment:
Certainly. Also, added it to another, similar spot which will likely need to wrap as well, which I'd missed.